src/screens/Ticketing/Tickets/RecentTickets/RecentTickets.tsx

```diff
@@ -45,7 +45,8 @@ export const RecentTickets = () => {
         if (
           ticketType === 'single' ||
           ticketType === 'period' ||
-          ticketType === 'carnet'
+          ticketType === 'carnet' ||
+          ticketType === 'hour24'
```

Comment:
Can't this be removed all together? 🤔  This should just be handled by the `productIsSellableInApp`.

Reply:
Or let me think. Maybe it is necessary when adding new products 🤔  Just leave it 😅

Reply:
yeah...if we add new products other than these. so not touching it for now :)